.gitops/charts/traffic-court-dev-values.yaml

```diff
@@ -7,14 +7,14 @@ global:
 
 arc-dispute-api:
   image:
-    tag: "1.44.4"
+    tag: "1.44.5"
```

Comment:
This is causing merge conflicts. In the future, I think it'd be better to separate release commits and application commits.  The changes in this file should be in a separate PR.